### ISSUE TITLE
Yarn Deploy Broke JS

### ISSUE THREAD

**Issue by katbmon** [Shopify/slate#639]
I recently did a yarn deploy to our Shopify site and when this occurred the js broke. We did not touch the src/js files, but the dist/js files were edited. 

However the JS is still working on local. Any ideas on how to fix the yarn deploy so the js is not broken on the live site?

**Comment by t-kelly** [comment]
Hey @katbmon -- I'm going to need a little more details if I can even start to help you. Can you please specify what has broken or what errors you are receiving? The description you provided is very generic.

**Comment by katbmon** [comment]
@t-kelly There was no errors output or issues with the deployment. The error is that on the live site the js is not longer loading after we yarn deploy to live. I also tried to yarn deploy the previous commit to undo the files, but that did not resolve the js issues on live after deploying. 

When I looked after the yarn deploy the js files in dist were changed but in src we did not change anything. An example of the changes are this:
Original: }, e.p = "", e(e.s = 214)
Changed: }, e.p = "", e(e.s = 213)

I am attaching another one as well:
<img width="1205" alt="screen shot 2018-06-20 at 10 15 24 am" src="https://user-images.githubusercontent.com/16745651/41664081-6bcee9c0-7494-11e8-897e-4c0ca7215643.png">

Is there any other information I could provide that would help? I appreciate your answer and helping me look into this.
